site2/docs/reference-configuration.md

```diff
@@ -817,6 +817,8 @@ The [Pulsar proxy](concepts-architecture-overview.md#pulsar-proxy) can be config
 |tokenAudienceClaim| The token audience "claim" name, e.g. "aud". It is used to get the audience from token. If it is not set, the audience is not verified. ||
 | tokenAudience | The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token need contains this parameter.| |
 |haProxyProtocolEnabled | Enable or disable the [HAProxy](http://www.haproxy.org/) protocol. |false|
+| numIOThreads | Number of threads to use for Netty IO. | 2 * Runtime.getRuntime().availableProcessors() |
```

Comment:
```suggestion
| numIOThreads | Number of threads used for Netty IO. | 2 * Runtime.getRuntime().availableProcessors() |
```
do you mean this?

Reply:
https://github.com/apache/pulsar/blob/c24a6615f7190615f54b2cbf6d2612488c96a33b/site2/docs/reference-configuration.md?plain=1#L486
I think we should use the same description.

Reply:
I think the description there is inaccurate.